README.md

```diff
@@ -1,7 +1,7 @@
-# portfolio-Mobile-Version
+# Mobile-Version
 
 
-# portfolio-Desktop-Version
+# Desktop-Version
```

Comment:
- [ ] The Readme is not descriptive at all. Let's add a descriptive Readme. 
- [ ] Please deploy your app to GitHub Pages and add the link to the Readme. By this, it will be easy for anyone to see your hard work. 
- [ ] I cannot find a recorded video for your project. Please work on it and add that to the Readme file.